src/components/ExecutorDetails/Columns.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { renderStatus, normalizeStatus } from '../statusHelper';
+import { inventoryUrlBuilder } from '../../Utilities/urls';
+
+const urlBuilder = inventoryUrlBuilder({ id: 'default' });
+
+export default [
+  {
+    key: 'display_name',
+    title: 'Name',
+    // eslint-disable-next-line
+    renderFunc: (name, id, { fqdn }) => <div><a href={urlBuilder(id)}>{fqdn || name || id}</a></div>
+  },
+  'tags',
```

Comment:
This value type (string) is somewhat different from what we had before (`state.columns.find((col) => col.key === 'tags')`, object?). I am not quite familiar with the app, and can't say it is bad or good, just suspicious.

Reply:
This is to consume the tags column from the Inventory app, rather than creating a custom. `mergeColumns` function is designed to handle this.